config/routes.rb

```diff
@@ -3,13 +3,25 @@
   root to: 'projects#index'
 
   namespace :api do
-    resources :deploys, only: [:index, :active_count] do
+    resources :deploys, only: [:active_count] do
       collection do
         get :active_count
       end
     end
 
-    resources :stages, only: [:index] do
+    resources :projects, param: :project_id, only: [] do
+      get :duplicable, on: :member, to: 'stages#duplicable'
```

Comment:
This needs the to: `stages#duplicable` because without it, the request routes to the projects controller.

See failure: https://travis-ci.org/zendesk/samson/jobs/151951746